File: truth_set/python/structure/organisation/statement-issues/calculateTotalPrice.py
def calculate_total_price(items, tax_rate):
    subtotal = None
    subtotal = sum(items)
    
    total = None 
    total = subtotal + subtotal * tax_rate
    return total

items = [29.99, 9.99, 4.99]
tax_rate = 0.07  # 7% tax
total_price = calculate_total_price(items, tax_rate)
print(f"Total Price: ${total_price:.2f}")

File: truth_set/go/structure/organisation/statement-issues/calculateTotalPrice.go
package main

import "fmt"

func calculateTotalPrice(items []float64, taxRate float64) float64 {
	var subtotal float64
	subtotal = 0
	for _, item := range items {
		subtotal += item
	}

	var total float64
	total = subtotal + subtotal*taxRate

	return total
}

func main() {
	items := []float64{29.99, 9.99, 4.99}
	taxRate := 0.07 // 7% tax

	var totalPrice float64
	totalPrice = calculateTotalPrice(items, taxRate)
	fmt.Printf("Total Price: $%.2f\n", totalPrice)
}
\ No newline at end of file